server.spec.js

```diff
@@ -0,0 +1,93 @@
+const server = require('./server.js');
+const request = require('supertest');
+
+describe('GET routing for /games', () => {
+    it('return status code 200', async () => {
+        const response = await request(server).get('/games')
+        expect(response.status).toBe(200);
+      
+    });
+    it('returns array of all games', async() => {
+        const response = await request(server).get('/games');
+        const expected = [
+            {
+                title: 'Final Fantasy VII',
+                genre: 'JRPG',
+                releaseYear: 1997
+            }
+                        ];
```

Comment:
This ending square bracket for this array ought to be at one less indent than the closing brace for the object inside of it.